maintainers/maintainer-list.nix

```diff
@@ -18358,6 +18358,12 @@
     github = "ymeister";
     githubId = 47071325;
   };
+  yoavlavi = {
+    email = "yoav@yoavlavi.com";
+    github = "yoav-lavi";
+    githubId = 14347895;
+    name = "Yoav Lavi";
+  };
```

Comment:
lgtm; In case you have a matrix account it would be nice if you added that as well there is quite active nixos matrix space at: https://matrix.to/#/#community:nixos.org

Reply:
I don't at the moment, I'm assuming I can add one after the fact if I ever create one?

Reply:
Sure!